fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2260,6 +2262,18 @@ The commit notification is sent in the request **PATCH /transfers/**_{ID}_ from
 
 **Figure 55 -- Commit notification where commit of transfer in Switch failed**
 
+In case of a validation failure (not schema validation) or another processing error during the processing of a **PUT /transfers/**_{ID}_ callback, a **PATCH /transfers/**_{ID}_**/error** notification is sent from the Switch to the Payee FSP to indicate of the failure. This notification is sent in both cases where the Payee FSP sends the _transferState_ as reserved or committed.
```

Comment:
The error notification _can_ be sent by a Switch. If it is sent or not should be up to scheme rules. The error notification should not be sent in case `RESERVED` is used as `transferState` in version 1.2, see https://github.com/mojaloop/mojaloop-specification/issues/103#issuecomment-970407712.

Reply:
Updated the wording here